pvmismatch/tests/test_pvmodule.py

```diff
@@ -1,38 +1,33 @@
 """
 Tests for pvmodules.
 """
-
-from nose.tools import ok_
+import pytest
```

Comment:
Last suggestion. I believe you can remove this import, b/c it's never used below. `pytest` is a test runner that replaces `nosetests` and is called from the command line to autodetect tests and run them. The only need to import pytest is if you want to use its features such as test fixtures, parameterization, catch exceptions, or mark skip tests. Check the [pytest docs](https://docs.pytest.org/en/latest/).

Reply:
@chetan201 PTAL